test/e2e/apimachinery/resource_quota.go

```diff
@@ -749,7 +829,15 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		err = waitForResourceQuota(f.ClientSet, f.Namespace.Name, resourceQuotaNotBestEffort.Name, usedResources)
 		framework.ExpectNoError(err)
 	})
-	ginkgo.It("Should be able to update and delete ResourceQuota.", func() {
+
+	/*
+		Release: v1.16
+		Testname: ResourceQuota, update and delete
+		Description: Create a ResourceQuota for CPU and Memory quota limits. Creation MUST be successful.
+		When ResourceQuota is updated to modify CPU and Memory quota limits, update MUST succeed with updated values for CPU and Memory limits.
+		When ResourceQuota is deleted, it MUST not be available in  namespace.
+	*/
+	framework.ConformanceIt("Should be able to update and delete ResourceQuota.", func() {
```

Comment:
Let's lowercase "Should", like all the others.

Reply:
Is it ok to reflect this change here in Promotion PR? Otherwise another PR needs to be raised for effective name change.

Reply:
yes, that's fine

Reply:
Done. Thanks